tests/test_framework/mod.rs

```diff
@@ -500,20 +502,21 @@ pub fn verify_swap_results(
             );
 
             assert!(
-                balances.regular == Amount::from_btc(0.14555884).unwrap() // First maker on successful coinswap
-                    || balances.regular == Amount::from_btc(0.14555886).unwrap() // First maker variant
-                    || balances.regular == Amount::from_btc(0.14555882).unwrap() // First maker variant Alternative
-                    || balances.regular == Amount::from_btc(0.14533014).unwrap() // Second maker on successful coinswap
-                    || balances.regular == Amount::from_btc(0.14533016).unwrap() // Second maker variant
+                balances.regular == Amount::from_btc(0.14555884).unwrap() // First maker
+                    || balances.regular == Amount::from_btc(0.14533014).unwrap() // Second maker
                     || balances.regular == Amount::from_btc(0.14999508).unwrap() // No spending
-                    || balances.regular == Amount::from_btc(0.14999510).unwrap() // No spending variant
+                    || balances.regular == Amount::from_btc(0.14999510).unwrap() // No spending variant (alternate transaction size +2 sats)
+                    || balances.regular == Amount::from_btc(0.14533016).unwrap() // maker alternative transaction size (+2 sats)
+                    || balances.regular == Amount::from_btc(0.14555886).unwrap() // maker alternative transaction size (+2 sats)
                     || balances.regular == Amount::from_btc(0.24999510).unwrap(), // Multi-taker scenario
                 "Maker seed balance mismatch"
             );
 
+            //TODO:Figure out why there is +-2 sats difference in regular and swap balance in some cases.
+
```

Comment:
Move this above the assert statements where the difference is actually happening.